app/models.py

```diff
@@ -6,7 +6,7 @@
 class Type(db.Model):
 
     id = db.Column(db.Integer, primary_key=True)
-    name = db.Column(db.String())  # name of type
+    name = db.Column(db.String(), default='individual')  # name of type
```

Comment:
Why are we giving Type a default?
```suggestion
    name = db.Column(db.String())  # Type doesn't need a default. The default type is for **Users**
```